.gitignore

```diff
@@ -1 +1,3 @@
 .ruby-version
+.tool-versions
+.idea
```

Comment:
FYI this is a good fit for your global gitignore too

Reply:
I do have it.

but I add it in case I helping a junior dev doing open-source.